src/components/Auth/Auth.js

```diff
@@ -3,8 +3,7 @@ import './auth.css';
 import Login from './Login/Login';
 import Register from './Register/Register';
 import { Redirect } from "react-router-dom";
-import Header from "../Header/Header";
-import Footer from "../Footer/Footer";
+import React from 'react';
 
 class Auth extends Component {
```

Comment:
As I know we decided to use the functional component approach in our React Application

Reply:
We also decided to refactor the approach after the demo.